lib/src/folding/foldable_block_type.dart

```diff
@@ -8,6 +8,9 @@ enum FoldableBlockType {
   /// ( )
   parentheses,
 
+  //just for python
+  spaces,
```

Comment:
Let's call this `indentation` because this can also originate from tabs (although we may indeed only support spaces for now).